specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/library.json

```diff
@@ -0,0 +1,479 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "ArtifactsClient"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/libraries": {
+      "get": {
+        "tags": [
+          "Library"
+        ],
+        "operationId": "Library_GetLibraryByWorkspace",
+        "x-ms-examples": {
+          "List Libraries": {
+            "$ref": "./examples/Library_ListByWorkspace.json"
+          }
+        },
+        "description": "Lists Library.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/LibraryListResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/libraries/{libraryName}/create": {
+      "put": {
+        "tags": [
+          "Library"
+        ],
+        "operationId": "Library_Create",
+        "x-ms-examples": {
+          "Create Library": {
+            "$ref": "./examples/Library_Create.json"
+          }
+        },
+        "description": "Create Library",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LibraryNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted."
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/libraries/{libraryName}/flush": {
+      "post": {
+        "tags": [
+          "Library"
+        ],
+        "operationId": "Library_Flush",
+        "x-ms-examples": {
+          "Flush Library": {
+            "$ref": "./examples/Library_Flush.json"
+          }
+        },
+        "description": "Flush Library",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LibraryNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted."
+          },
```

Comment:
same question on whether this is a long running operation, if so mark it as lro.

Reply:
same question for patch api.